scripts/check_if_file_indented.sh

```diff
@@ -20,41 +20,11 @@
 # along with t8code; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
-# We use clang-format to indent our code. There are different base styles
-# available, but we use a modification. The options for this modification
-# are located in the .clang-format file.
-# We have to use the --dry-run argument, so that clang-format does not change
-# anything. -Werror produces an error if the file is not indented cerrctly.
-# The --style=file arguments tells clang-format to look for a *.clang-format file.
-FORMAT_OPTIONS="--dry-run --Werror --style=file"
+GIT_REPO_PATH=$(git rev-parse --show-toplevel)
```

Comment:
Would be nice to still have a brief description of what the script can be used for here.